discord/abc.py

```diff
@@ -1345,7 +1345,7 @@ async def send(
         file: File = ...,
         stickers: Sequence[GuildSticker | StickerItem] = ...,
         delete_after: float = ...,
-        nonce: str | int = ...,
+        nonce: int | str | None = ...,
```

Comment:
Why optional?

https://discord.com/developers/docs/resources/channel#create-message-jsonform-params

Reply:
It appears to be documented as optional in what you linked

Reply:
yeah it's optional in docs

Reply:
Well, it's optional in the sense that it might not be present, but what I meant is it can't be None i.e. it's not nullable.

Reply:
> Well, it's optional in the sense that it might not be present, but what I meant is it can't be None i.e. it's not nullable.

hmm tbh makes sense here, ill just leave it like that in http